LibreNMS/Util/Version.php

```diff
@@ -69,12 +70,26 @@ public function local(): string
     public function localCommit(): array
     {
         if ($this->is_git_install) {
-            [$local_sha, $local_date] = array_pad(explode('|', rtrim(`git show --pretty='%H|%ct' -s HEAD`)), 2, '');
+            $install_dir = base_path();
+            $version_process = new Process(['git', 'show', '-q', '--pretty=%H|%ct'], $install_dir);
+            $version_process->run();
+
+            // failed due to permissions issue
+            if ($version_process->getExitCode() == 128 && Str::startsWith($version_process->getErrorOutput(), 'fatal: unsafe repository')) {
+                (new Process(['git', 'config', '--global', '--add', 'safe.directory', $install_dir]))->run();
```

Comment:
I wonder if we should do this, or just suggest it instead

Reply:
But who cares ä

Reply:
Maybe when the validation fixer is merged :D